src/loaders/ncbi_downloader/gtdb.py

```diff
@@ -0,0 +1,297 @@
+"""
+usage: gtdb.py [-h] --release_ver {202,207,214,80,83,86,89,95} [--root_dir ROOT_DIR]
```

Comment:
The markdown documentation will need to be updated as well.

Reply:
Hi, may I know exactly which part I need to update here?

Reply:
Anything refers to the old ncbi script which now doesn't exist, and the replacement has different arguments

Reply:
Updated! But please let me know if I missing anything again:)

Reply:
You're probably more familiar with the markdown documentation than I am at this point.

That being said, there's no changes to the markdown docs in the PR AFAICT

Reply:
Updated markdown. Also add env that's missing

Reply:
:+1: